src/test/java/org/minions/devfund/danielmontecinos/moviedatabase/MovieTest.java

```diff
@@ -0,0 +1,87 @@
+package org.minions.devfund.danielmontecinos.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests Movie class.
+ */
+public class MovieTest {
+    private Movie movie;
+
+    /**
+     * Creates movie object.
+     */
+    @Before
+    public void setUp() {
+        movie = new Movie();
+    }
+
+    /**
+     * Tests getName method.
+     */
+    @Test
+    public void testGetNameMethod() {
+        assertNull(movie.getName());
+    }
+
+    /**
+     * Tests testsSetNameMethod method.
+     */
+    @Test
+    public void testSetNameMethod() {
+        movie.setName("Movie 1");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Movie 1" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)